### PR TITLE
Project Execution - P1

### PR DESCRIPTION
**Pull request by McDoxa**

### CODE REVIEW

File: project-javascript-p1-aync-await/Oluwaseyi.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+<body>
+    <script>
+    const currentTime = 21; //new Date().getHours();
+    const currentDate = 5 //new Date().getMonth();
+    async function smartTurnOffLight(){
+        return new Promise((resolve, reject) =>{
+            if(currentTime > 7 && currentTime < 18){
+                setTimeout(()=>{
+                    resolve('Turn Off Light Done!')},3000);
+                        } else {
```
Comment (basir) [inline project-javascript-p1-aync-await/Oluwaseyi.html]: line indent is wrong.

File: project-javascript-p1-aync-await/Oluwaseyi.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            console.log('Energy Saving is ON!')
+        } catch(error) {
+            console.log(error);
+            }
+    };
+    energySaving();
+    /* Concurrency for Independent promises */
+    const energySavingConcurrent = async() => {
+        try{
+            const resultArray = await Promise.all([smartTurnOffLight(),smartTurnOffAirConditioner(),smartTurnOffWashingMachine()])
```
Comment (basir) [inline project-javascript-p1-aync-await/Oluwaseyi.html]: you could use array deconstruction to have result in 3 variables.

File: project-javascript-p1-aync-await/Oluwaseyi.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                }
+        );
+    }
+    /* Dependent promises */
+   const energySaving = async() => {
+        try{
+            const firstValue = await smartTurnOffLight();
+            const secondValue = await smartTurnOffAirConditioner();
+            const thirdValue = await smartTurnOffWashingMachine();
+            console.log('Energy Saving is ON!')
```
Comment (basir) [inline project-javascript-p1-aync-await/Oluwaseyi.html]: Nice

File: project-javascript-p1-aync-await/Oluwaseyi.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                            setTimeout(()=>{
+                                reject('Failed to Turn Off Air Conditioner')},3000);
+                                }
+                }
+        );
+    }
+
+    async function smartTurnOffWashingMachine(){
+        return new Promise((resolve, reject) =>{
+            if(currentTime > 20 && currentTime < 22){
```
Comment (basir) [inline project-javascript-p1-aync-await/Oluwaseyi.html]: change this condition to cover  2pm to 5pm

**Comment by McDoxa** [comment]
Reviewed changes have been effected.

**Pull request merged**